Gemfile

```diff
@@ -1,11 +1,11 @@
-# encoding: utf-8
 source 'https://rubygems.org'
 
 gemspec
 
 group :development do
   gem 'bundler'
   gem 'minitest'
+  gem 'pry'
   gem 'rake'
   gem 'rubocop', '= 0.49.1'
```

Comment:
Please :pray: add version constraints to all gems listed here, latest installed is fine, and use the pessimistic operator to allow patch version updates. For example: `gem 'rubocop', '~> 0.49.1'`.

Reply:
Added